man/systemd-vmspawn.xml

```diff
@@ -361,6 +353,42 @@
       </variablelist>
     </refsect2>
 
+    <refsect2>
+      <title>Input/Output Options</title>
+
+      <variablelist>
+        <varlistentry>
+          <term><option>--console=</option><replaceable>MODE</replaceable></term>
+
+          <listitem><para>Configures how to set up the console of the VM. Takes one of
+          <option>interactive</option>, <option>read-only</option>, <option>native</option>,
+          <option>gui</option>. Defaults to <option>interactive</option>. <option>interactive</option>
+          provides an interactive terminal interface to the VM. <option>read-only</option> is similar, but is
```

Comment:
Hmm, we usually use `<literal>` for possible values of an option?

Reply:
I'd say, it's a bit chaotic what we do there... sometimes this and sometimes that ;-9

but i changed it to <literal> now